app/routes/action.js

```diff
@@ -116,5 +126,66 @@ module.exports = [
         })
       return h.response(filteredActions).code(OK_STATUS_CODE)
     }
+  },
+  {
+    method: 'POST',
+    path: '/action/{pathParam}/rule/',
+    options: {
+      validate: {
+        payload: Joi.object({
+          id: Joi.string().required(),
+          config: Joi.object().optional()
+        })
+      }
+    },
+    handler: (request, h) => commonHandler(request, h, (action, request, h) => {
+      const newRule = request.payload
+      addRule(action, newRule)
+      console.log(JSON.stringify(getActions(), null, 2))
```

Comment:
The console.log lines can be removed in this file

Reply:
removed